odxtools/odxlink.py

```diff
@@ -175,6 +176,9 @@ def resolve(self, ref: OdxLinkRef) -> Any:
 
             obj = doc_frag_db.get(odx_id)
             if obj is not None:
+                if odx_id.local_id == "stupid.sdg.caption":
+                    self._stupid_sdg_resolved = True
```

Comment:
what?

Reply:
whoops. that is an artifact of the instrumentation code which I used to develop this: I added an SDG which contained an SDG-caption with the ID "stupid.sdg.caption" to the XML tag I wanted to test and ensured that it was seen during the ID resolution. (using this method I found a few classes which had issues with ID resolution, BTW.)